### PR TITLE
Add list of VMs

### PR DESCRIPTION
**Pull request by cgcardona**
1. Format for markdownlint
2. Add golang and rustlang sdks

### CODE REVIEW

File: docs/subnets/README.md [hunk 1]
```diff
@@ -1,73 +1,151 @@
...
+
+### Example Virtual Machines
+
+The following VMs were built using Go
+
+- [SubnetEVM](https://github.com/ava-labs/subnet-evm/)
+- [SpacesVM](https://github.com/ava-labs/spacesvm)
+- [BlobVM](https://github.com/ava-labs/blobvm)
+- [TimestampVM](https://github.com/ava-labs/timestampvm)
+
```
Comment (yulin-dong) [inline docs/subnets/README.md:128]: We should follow and update the project name convention in https://github.com/ava-labs/avalanche-docs/blob/master/style-guide.md

File: docs/subnets/README.md [hunk 2]
```diff
@@ -1,73 +1,151 @@
...
+- [How to Build a Simple Golang VM](https://docs.avax.network/subnets/create-a-vm-timestampvm)
+- [How to Build a Complex Golang VM](https://docs.avax.network/subnets/create-a-vm-blobvm)
+
+The following VMs were built using Rust via the [Avalanche RustSDK](https://crates.io/crates/avalanche-types)
+
+- [TimestampVM](https://github.com/ava-labs/timestampvm-rs)
+- [SpacesVM](https://github.com/ava-labs/spacesvm-rs)
+- [BulletproofVM](https://github.com/usmaneth/BulletproofVM)
+
```
Comment (yulin-dong) [inline docs/subnets/README.md:139]: We should also distinguish VM in different languages, https://docs.avax.network/subnets/create-a-simple-rust-vm call it `TimestampVM RS`.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/subnets/README.md [hunk 3]
```diff
@@ -1,73 +1,151 @@
...
-Avalanche VMs (AVMs) make it easy to define a blockchain-based decentralized application. Rather than new, limited languages like Solidity, developers can write VMs in Go (other languages will be supported in the future).
+Avalanche VMs (AVMs) make it easy to define a blockchain-based decentralized
+application. Rather than new, limited languages like Solidity, developers can
+write VMs in Go (other languages will be supported in the future).
+
+### Example Virtual Machines
+
+The following VMs were built using Go
+
+- [SubnetEVM](https://github.com/ava-labs/subnet-evm/)
```
Comment (yulin-dong) [inline docs/subnets/README.md:124]: We should add Coreth here as it is a VM too.

File: docs/subnets/README.md [hunk 4]
```diff
@@ -1,73 +1,151 @@
...
+programmers.
 
-Avalanche VMs (AVMs) make it easy to define a blockchain-based decentralized application. Rather than new, limited languages like Solidity, developers can write VMs in Go (other languages will be supported in the future).
+Avalanche VMs (AVMs) make it easy to define a blockchain-based decentralized
+application. Rather than new, limited languages like Solidity, developers can
+write VMs in Go (other languages will be supported in the future).
+
+### Example Virtual Machines
+
+The following VMs were built using Go
```
Comment (yulin-dong) [inline docs/subnets/README.md:122]: We should add Coreth here as it is a VM too.

File: docs/subnets/README.md [hunk 5]
```diff
@@ -1,73 +1,151 @@
...
-Ethereum made a step toward solving this problem with smart contracts. Developers didn’t need to worry about networking and consensus, but creating decentralized applications was still hard. The Ethereum VM has low performance and imposes restrictions on smart contract developers. Solidity and the other few languages for writing Ethereum smart contracts are unfamiliar to most programmers.
+Ethereum made a step toward solving this problem with smart contracts.
+Developers didn’t need to worry about networking and consensus, but creating
+decentralized applications was still hard. The Ethereum VM has low performance
+and imposes restrictions on smart contract developers. Solidity and the other
+few languages for writing Ethereum smart contracts are unfamiliar to most
+programmers.
 
-Avalanche VMs (AVMs) make it easy to define a blockchain-based decentralized application. Rather than new, limited languages like Solidity, developers can write VMs in Go (other languages will be supported in the future).
+Avalanche VMs (AVMs) make it easy to define a blockchain-based decentralized
```
Comment (yulin-dong) [inline docs/subnets/README.md:116]: Not sure if this sentence should be refactored or removed, as we don't mention AVM anywhere else.

File: docs/subnets/README.md (hunk 2 above)
Comment (cgcardona) [reply docs/subnets/README.md:141]: Resolved

File: docs/subnets/README.md (hunk 5 above)
Comment (cgcardona) [reply docs/subnets/README.md]: Resolved

File: docs/subnets/README.md (hunk 4 above)
Comment (cgcardona) [reply docs/subnets/README.md:122]: Resolved

File: docs/subnets/README.md (hunk 1 above)
Comment (cgcardona) [reply docs/subnets/README.md:130]: Resolved

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**